sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2472,121 +2495,80 @@ object Radar {
                     return
                 }
 
-                sendEvent(customType, metadata, location, events, user, callback)
+                sendLogConversionRequest(conversionType, metadata, callback)
```

Comment:
open q: if the track call fails, should we still be catching that and sending the conversion event?